blueprints/automation/elegoo_printer/elegoo_printer_progress.yaml

```diff
@@ -123,24 +179,193 @@ action:
+          - condition: template
+            value_template: >-
+              {{ states(error_status_reason_entity) not in ['ok', 'none', 'unknown', 'unavailable'] }}
+        sequence:
+          - device_id: !input notify_device
+            domain: mobile_app
+            type: notify
+            title: "🚨 Critical Printer Error!"
+            message: >-
+              {% set error = states(error_status_reason_entity) %}
+              {% set file_name = states(file_name_entity)|default('current print', true) %}
+              {% if error == 'filament_runout' %}
+                🧵 Filament runout detected during {{ file_name }}! Please load new filament.
+              {% elif error == 'filament_jam' %}
+                🚫 Filament jam detected during {{ file_name }}! Please check the extruder.
+              {% elif error == 'temp_error' %}
+                🌡️ Temperature error detected! Please check nozzle and bed temperatures.
+              {% elif error == 'level_failed' %}
+                📐 Bed leveling failed! Please check the bed leveling system.
+              {% elif error == 'home_failed' or error == 'home_failed_x' or error == 'home_failed_y' or error == 'home_failed_z' %}
+                🏠 Homing failed! Please check the printer axes and endstops.
+              {% elif error == 'bed_adhesion_failed' %}
+                🛏️ Print detached from bed during {{ file_name }}!
+              {% elif error == 'move_abnormal' %}
+                ⚙️ Motor movement abnormality detected!
+              {% elif error == 'file_error' %}
+                📁 Print file error during {{ file_name }}!
+              {% elif error == 'udisk_remove' %}
+                💾 USB drive was removed during printing!
+              {% elif error == 'nozzle_temp_sensor_offline' %}
+                🌡️ Nozzle temperature sensor is offline!
+              {% elif error == 'bed_temp_sensor_offline' %}
+                🌡️ Bed temperature sensor is offline!
+              {% elif error == 'camera_error' %}
+                📷 Camera connection error!
+              {% elif error == 'network_error' %}
+                🌐 Network connection error!
+              {% elif error == 'server_connect_failed' %}
+                🖥️ Server connection failed!
+              {% elif error == 'disconnect_app' %}
+                📱 Controlling app disconnected during print!
+              {% else %}
+                ⚠️ Error: {{ error }}
+              {% endif %}
+            data:
+              image: "{{ ('/api/camera_proxy/' ~ camera_entity) if (camera_entity|default('', true)) != '' else '' }}"
+              url: "{{ dashboard_url }}"
+              clickAction: "{{ dashboard_url }}"
+              group: "{{ notification_group }}"
+              channel: "{{ notification_group }}"
+              tag: "{{ notification_group }}_critical_error"
+              sticky: true
+              alert_once: false
+              push:
+                interruption-level: critical
```

Comment:
for me I'd want to tune this down so it'd be nice if it's at least configurable - critical on ios circumvents things like do not disturb/silent mode which is reserved for pretty extreme things - and I don't put my printer in that category :-)

Reply:
good call.. fixed

Reply:
done